rds-replica/variables.tf

```diff
@@ -62,6 +62,17 @@ variable "storage_encrypted" {
   default     = true
 }
 
+variable "storage" {
+  description = "How many GBs of space does your database need?"
+  default     = "10"
```

Comment:
Default should be `null` to avoid changing the current default behaviour. If not set, it should default to the allocated storage of the replicated instance.